src/types/emr/specimenDefinition/specimenDefinition.ts

```diff
@@ -30,9 +30,9 @@ export interface DurationSpec {
 
 export interface ContainerSpec {
   description: string | null;
```

Comment:
```suggestion
  description?: string | null;
```

<img width="670" alt="image" src="https://github.com/user-attachments/assets/d222b298-ba61-4222-8519-7a3f90098c68" />